src/components/ProgramRecord/ProgramRecord.jsx

```diff
@@ -143,7 +150,7 @@ function ProgramRecord({ isPublic }) {
           isOpen={sendRecord.sendRecordModalOpen}
           toggleSendRecordModal={toggleSendRecordModal}
           creditPathways={recordDetails.record.pathways}
-          uuid={programId}
+          programId={programId}
```

Comment:
Is this `programId` the auto-generated (integer) identifier from our database of the Program? or is this the program's UUID? I think, if this is the UUID (which is the assumption I'm getting from the code), I think `programId` _really_ should be something like `programUUID`.

Reply:
This is the UUID. I wasn't aware of another program identifier other than the UUID. I'll change this all around to make it clear 👍

Reply:
Thanks, Max. I think making it clear that this is a `programUUID` would really help clear things up here.